### ISSUE TITLE
dataset directory

### ISSUE THREAD

**Issue by xins981** [yzqin/s4g-release#12]
@yzqin, hi yuzhe. can you provide the full dataset directory structure? 

similar below:
![image](https://github.com/yzqin/s4g-release/assets/92990192/13601506-84b9-46ab-b8a5-ae8d7bc9d38c)

Although I clear up the data_gen code, it's not easy gen training data, I‘sure I'm not alone. this is barrier for promoting this outstanding work. if possible, hope you providing more training details. 
thank you and your good job!

**Comment by yzqin** [comment]
hi @xins981 

Thank you for your interest in our project. I apologize for any inconvenience, but this work was completed in 2019, which is now 4 years ago. Due to the large size of the dataset file, I had to remove the entire dataset directory, retaining only the code that generates them. The code available here is identical to the one used to generate the training data for the pre-trained model in CoRL 2019.

I regret any trouble this might have caused. After this project, I shifted my research focus and have not worked on grasping detection since, so I do not have any subsequent work related to this paper.

If you have specific technical questions about the code, I'll be more than happy to provide information, to the best of my recollection.
